### PR TITLE
[ntuple] Various smaller improvements

### PR DESCRIPTION
**Pull request by jblomer**
Preparation for the change set implementing the RNTuple merger (as sketched in #6105)

### CODE REVIEW

File: tree/ntuple/v7/src/RNTupleDescriptorFmt.cxx [hunk 1]
```diff
@@ -113,8 +113,8 @@ void ROOT::Experimental::RNTupleDescriptor::PrintInfo(std::ostream &output) cons
    std::uint64_t nPages = 0;
    int compression = -1;
    for (const auto &column : fColumnDescriptors) {
-      auto element = Detail::RColumnElementBase::Generate(column.second.GetModel().GetType());
-      auto elementSize = element.GetSize();
+      auto elementSize = Detail::RColumnElementBase::Generate(
```
Comment (pcanal) [inline tree/ntuple/v7/src/RNTupleDescriptorFmt.cxx:116]: I am confused ... maybe ...   Why do we need to make an allocation just to get the element size?

**Review by pcanal — APPROVED** [review]
This looks good ... with just a question that is really about the pre-existing code.

File: tree/ntuple/v7/src/RNTupleDescriptorFmt.cxx (hunk 1 above)
Comment (jblomer) [reply tree/ntuple/v7/src/RNTupleDescriptorFmt.cxx:116]: That's not great indeed.  We could define a static array that stores the on-disk sizes of an element of a given [column type](https://github.com/root-project/root/blob/master/tree/ntuple/v7/inc/ROOT/RColumnModel.hxx#L35). Or is this too brittle?

Comment (pcanal) [reply tree/ntuple/v7/src/RNTupleDescriptorFmt.cxx:116]: I guess you could also duplicate the switch statement that is already in `RColumnElementBase::Generate`

**Pull request merged**